libraries/AP_NavEKF2/AP_NavEKF2.h

```diff
@@ -493,6 +493,28 @@ class NavEKF2 {
     // time of last lane switch
     uint32_t lastLaneSwitch_ms;
 
+    enum class INIT_FAILURES {
```

Comment:
It seems in the EKF the convention for enums is to use camel case.  So perhaps it should be "InitFailures"